internal/daemon/controller/handlers/targets/target_service.go

```diff
@@ -1232,6 +1322,32 @@ func (s Service) getFromRepo(ctx context.Context, id string) (target.Target, []t
 	return u, hs, cl, nil
 }
 
+// resolveAlias replaces the value of in with the destination id of the alias if
```

Comment:
Subbing out "value" for "in" might improve readability here
```suggestion
// resolveAlias returns the alias associated with in if the alias is
```